audit/projects/k8s-artifacts-prod/iam.json

```diff
@@ -42,7 +42,7 @@
     },
     {
       "members": [
-        "serviceAccount:k8s-infra-gcr-auditor@k8s-artifacts-prod.iam.gserviceaccount.com"
+        "deleted:serviceAccount:k8s-infra-gcr-auditor@k8s-artifacts-prod.iam.gserviceaccount.com?uid=111422293292441494221"
```

Comment:
Yikes!  With great power...

Reply:
Yeah this was me https://github.com/kubernetes/k8s.io/issues/1730